google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/stub/metrics/BigtableMetricsRecorder.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.cloud.bigtable.data.v2.stub.metrics;
+
+import io.opentelemetry.api.common.Attributes;
+
+/** A helper class to record Bigtable metrics. */
+class BigtableMetricsRecorder {
+
+  void recordOperationLatencies(long value, Attributes attributes) {}
```

Comment:
Can we change the parameter type from `Attributes` to `Map<String, Object>` or `Map<String, String>` so that this class is agnostic of OpenTelemetry?

Reply:
Good idea!